source/tests/system/nirfmxwlan_driver_api_tests.cpp

```diff
@@ -185,8 +185,8 @@ TEST_F(NiRFmxWLANDriverApiTests, OFDMModAccTXPCompositeFromExample_FetchData_Dat
   const auto txp_fetch_measurement_response = client::txp_fetch_measurement(stub(), session, "", 10.0);
   EXPECT_WARNING(RISING_EDGE_DETECTION_FAILED_WARNING, RISING_EDGE_DETECTION_FAILED_WARNING_STR, session, txp_fetch_measurement_response);
 
-  EXPECT_LT(0.0, ofdm_mod_acc_fetch_composite_rmsevm_response.composite_rms_evm_mean());
-  EXPECT_LT(0.0, ofdm_mod_acc_fetch_composite_rmsevm_response.composite_data_rms_evm_mean());
+  EXPECT_NE(0.0, ofdm_mod_acc_fetch_composite_rmsevm_response.composite_rms_evm_mean());
+  EXPECT_NE(0.0, ofdm_mod_acc_fetch_composite_rmsevm_response.composite_data_rms_evm_mean());
```

Comment:
What caused a previously passing test for less-than to now fail? Was it just an invalid assumption with a ticking time bomb?

Reply:
When we wrote the RFmx tests we didn't plan ahead of time what the expected output values _ought to_ be, we just looked at the actual value returned by the simulated device, decided it was reasonable, and wrote the test to watch for similar values. Sometimes, like in this case, the value started out less than or greater than zero during our initial testing, but further testing showed it to be sometimes the opposite. Mostly, though, we're just trying to verify that we can query through gRPC to get _some_ sort of value without it erroring. The _actual_ values should be tested by the tests for the actual driver.